scripts/get_famous_people_list/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6
+
+COPY ./get_famous_people_list/requirements.txt /app/requirements.txt
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir -r /app/requirements.txt
+RUN python -m spacy download en_core_web_lg
```

Comment:
Consider merging the three RUN layers so that you minimize the number of docker layers and therewith image size. See tip 3 here: https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34

Reply:
fixed!